on_the_porch/unused/data_download/download_crime_data.py

```
@@ -0,0 +1,180 @@
+                print("✅ Reached end of data")
+                break
+                
+            offset += limit
+            
+            # Be nice to the API
+            time.sleep(1)
+            
+        except requests.exceptions.RequestException as e:
+            print(f"❌ Request failed: {e}")
+            break
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)